api/api.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	"github.com/labstack/echo"
+	"github.com/wimpykid26/hermes/models"
```

Comment:
This should be `github.com/osdc/hermes/models`, I'll give you access to make branch.